### PR TITLE
Create timeseries example for time index

### PR DESCRIPTION
**Pull request by brechtvdv**

### CODE REVIEW

File: examples/timeseries.ttl [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@prefix tree: <https://w3id.org/tree#>.
+@prefix hydra: <http://www.w3.org/ns/hydra/core#>.
+@prefix dcterms: <http://purl.org/dc/terms/>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix schema: <http://schema.org/>.
+@prefix shacl: <http://www.w3.org/ns/shacl#>.
+@prefix lc: <http://semweb.mmlab.be/ns/linkedconnections#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix sosa: "http://www.w3.org/ns/sosa/".
```
Comment (pietercolpaert) [inline examples/timeseries.ttl]: `<>` not `""`

File: examples/timeseries.ttl [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+<https://example.org/observations> a hydra:Collection ;
+                                          rdfs:label "All observations we have."@en ;
+                                          tree:view </14/8393/5467?2019-07-23T00:00:00.000Z.ttl> . #From the latest sensor page, all elements of one tile can be found through "previous"
```
Comment (pietercolpaert) [inline examples/timeseries.ttl]: `;` not `.` at the end: parse error

**Comment by brechtvdv** [comment]
Turtle validator gives error with hydra:required true?

**Comment by pietercolpaert** [comment]
You cannot end with a period in a blank node

**Comment by pietercolpaert** [comment]
Indenting and coordinates

**Comment by brechtvdv** [comment]
add tree:timeQuery

**Comment by brechtvdv** [comment]
@pietercolpaert Should we add a tree:view relation for every tile that contains data? Looks like a good discovery way for harvesting all the data without having to guess through the template

**Comment by pietercolpaert** [comment]
Yes, otherwise you cannot know that this page is part of the collection. There are 3 options. See discovery spec. I think tree:view here makes sense, as long as you can discover all elements in the collection from that node.

File: examples/timeseries/14/8393/5467/2020-12-25T150000.ttl [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix sosa: <http://www.w3.org/ns/sosa/>.
+@prefix tiles: <https://w3id.org/tree#> .
+@prefix geo: <http://www.w3.org/2003/01/geo/wgs84_pos#> .
+@prefix cot: <https://w3id.org/city_of_things#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+
+</14/8393/5467?page=2020-12-25T150000.ttl> a tree:Node ;
+                                          dcterms:isPartOf <https://mycity.example.org/airQualityObservations> ;
+                                          tree:relation [     a tree:LowerThanRelation ;
```
Comment (pietercolpaert) [inline examples/timeseries/14/8393/5467/2020-12-25T150000.ttl:15]: LessThanRelation

File: examples/timeseries/14/8393/5467/2020-12-25T150000.ttl [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+_:b1 a sosa:Observation;
+  sosa:resultTime "2020-12-25T15:05:00.000Z"^^xsd:dateTime ;
+  sosa:hasFeatureOfInterest <#Airquality/lat=51.23759466223419&long=4.415932092815638>, cot:AirQuality ; 
+  sosa:observedProperty cot:NO2 .
+
+_:b2 a sosa:Observation ;
+  sosa:resultTime "2020-12-25T15:10:00.000Z"^^xsd:dateTime ;
+  sosa:hasFeatureOfInterest <#Airquality/lat=51.23759466223419&long=4.415932092815638>, cot:AirQuality ;
+  sosa:observedProperty cot:PM10 .
+
```
Comment (pietercolpaert) [inline examples/timeseries/14/8393/5467/2020-12-25T150000.ttl:48]: No search form on this page?

File: examples/timeseries/airQualityObservations.ttl [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix lc: <http://semweb.mmlab.be/ns/linkedconnections#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix sosa: <http://www.w3.org/ns/sosa/>.
+@prefix tiles: <https://w3id.org/tree#> .
+@prefix geo: <http://www.w3.org/2003/01/geo/wgs84_pos#> .
+@prefix cot: <https://w3id.org/city_of_things#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+
+<https://mycity.example.org/airQualityObservations> a hydra:Collection ;
```
Comment (pietercolpaert) [inline examples/timeseries/airQualityObservations.ttl:13]: Let’s change this to a tree:Collection

File: examples/timeseries/airQualityObservations.ttl [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+_:b0  a  hydra:IriTemplate ;
+      hydra:variableRepresentation hydra:BasicRepresentation ;
+      hydra:template "https://lodi.ilabt.imec.be/air/{z}/{x}/{y}{?page}";
+      hydra:mapping [
+        a hydra:IriTemplateMapping ;
+        hydra:variable "x" ;
+        tree:path (sosa:hasFeatureOfInterest geo:longitude)  ;
+        hydra:property tiles:longitudeTile ;
+        hydra:required "true"^^xsd:boolean ;
+        sh:minInclusive "8393" ;
```
Comment (pietercolpaert) [inline examples/timeseries/airQualityObservations.ttl:28]: This is really interesting! Didn’t think of this yet and it is not yet in the spec, but we could include this indeed!

Comment (pietercolpaert) [reply examples/timeseries/airQualityObservations.ttl:28]: Can you make an issue to add this to the spec?

**Review by pietercolpaert — CHANGES_REQUESTED** [review]
Sorry for the late reaction! Couple of points of feedback still!

File: examples/timeseries/airQualityObservations.ttl [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+@prefix tiles: <https://w3id.org/tree#> .
+@prefix geo: <http://www.w3.org/2003/01/geo/wgs84_pos#> .
+@prefix cot: <https://w3id.org/city_of_things#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+
+<https://mycity.example.org/airQualityObservations> a hydra:Collection ;
+                                                  rdfs:label "All airquality observations of our city."@en ;
+                                                  # Point to high-level nodes (per tile)
+                                                  void:subset </14/8393/5467> ; # This will redirect to the page with the latest sensor data from that tile, all observations within one tile can be found by following the "LowerThanRelation" recursively
+                                                  hydra:search _:b0 .
```
Comment (pietercolpaert) [inline examples/timeseries/airQualityObservations.ttl:17]: We’ll change this to `tree:search`!

**Review by pietercolpaert — APPROVED** [review]

**Comment by pietercolpaert** [comment]
Yes, thanks for this!

**Pull request merged**